charts/gardener-dashboard/charts/runtime/templates/dashboard/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.global.dashboard.enabled }}
```

Comment:
add condition to only render if ingress.hosts is set, otherwise it will fail anyhow. E.g. for the local kind deployment an ingress is not really required